docs/guide/catalog/catalog_configuration.md

```diff
@@ -24,12 +24,15 @@ ibexa_product_catalog:
         default:
             type: local
             options:
+                ibexa_product_catalog_root: 12
```

Comment:
This example is invalid. Correct option name is `root_location_remote_id` and as name suggests it expect remote id.